File: generateIndex.rs
use std::fs;
fn main() { 
    let c = ["TaskList", "PropsToCompExample", "ConditionalRenderingExample", "RenderingListsExample", "PureComponent"]
    fs::write("./src/app/index.ts", c.iter().map(|comp| format!("export {{ default as {} }} from './{}';", comp, comp)).collect::<String>())
}

File: generateIndex.cpp
# include <fstream>
int main() {
    for (const char* comp: {"TaskList", "PropsToCompExample", "ConditionalRenderingExample", "RenderingListsExample", "PureComponent"})
    std::ofstream("./src/app/index.ts", std::ios_base::app) << "export { default as " << comp << " } from './'" << comp << ";";
}